package.json

```diff
@@ -33,17 +33,18 @@
   "scripts": {
     "dev": "nodemon",
     "start": "yarn build && cd build && node server.js",
-    "build": "rimraf ./build && tsc && rsync -a --exclude \"*/node_modules\" ./src/plugins ./build",
+    "build": "./utils/build_all.sh",
     "test": "jest --runInBand",
     "pg": "exec ./utils/postgraphile_launch.sh",
     "pg_permissions": "exec ./utils/postgraphile_launch_permissions.sh",
     "dev_pg": "concurrently --kill-others-on-fail \"yarn dev\"  \"yarn pg\"",
     "dev_pg_permissions": "concurrently --kill-others-on-fail \"yarn dev\"  \"yarn pg_permissions\"",
-    "database_init": "./database/initialise_database.sh && ./database/insert_data.sh && ./database/post_data_insert.sh",
-    "build_plugins": "exec ./utils/build_plugins.sh",
+    "database_init": "exec ./database/initialise_database.sh",
+    "build_plugins": "exec ./utils/pluginScripts/build_plugins.sh",
```

Comment:
This file doesn't exist. Did you perhaps mean `build_all.sh`?

Reply:
oops, thanks for noticing:

396a243682d04d1cef5d6209c85397ad70ed808e